_Sidebar.md

```diff
@@ -9,7 +9,7 @@
     * [[Voice artists | Instructions-for-voice-artists]]
     * [[Designers and artists|Contributing-to-Oppia's-design]]
   * Reporting a bug
-    * [[File an issue for bug|Github-Issue-Guide]]
+    * [[How to File a Bug Report|Github-Issue-Guide]]
```

Comment:
Why does this need to be in a sub-bullet point vs at the top-level?

Also should the page name be renamed to match the link title? "GitHub Issue Guide" is a bit vague and if the link and destination don't match, users may think they navigated to the wrong place.

Reply:
I have made it as top-level point now and also renamed it to match the link title. Thanks !